Hashicorp Vault cluster health

alexandrumarian-portal opened the pull request:
@samber : Hashicorp Vault cluster health alert

samber commented:
Hi @alexandrumarian-portal and thanks for this new contribution

I'm not sure using the `up{...}` series is a good idea because your rule will check for scraping status instead of the real cluster health.

Did you test something like `sum(vault_core_active) / count(vault_core_active) > 0.5` ?

Doc: https://developer.hashicorp.com/vault/docs/internals/telemetry

Please write your rules in `_data/rules.yml`. `dist/*` files are generated.

alexandrumarian-portal commented:
Hi @samber ,

thank you for pointing it and thanks a lot for your suggestion.

I have deleted the entries in dist/* and added the rule in _data/rules.yml . 

Thank you.
